client/schema/v0/index.js

```diff
@@ -203,8 +203,9 @@ export default () => ({
           {
             name: 'purpose',
             label: 'Which purposes apply to your project?',
-            type: 'checkbox',
+            type: 'permissible-purpose',
             className: 'smaller',
+            preserveHeirarchy: true,
```

Comment:
How did this ever get through?!

`preserveHierarchy` damnit!

Reply:
HEHE